Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mhart/alpine-node
```

Comment:
I'd prefer to use the latest LTS release and the smaller base image (excludes NPM), which is currently `mhart/alpine-node:base-6`